coalib/settings/SectionFilling.py

```diff
@@ -61,6 +61,7 @@ def fill_section(section, acquire_settings, log_printer, bears):
 
 
 def fill_settings(sections,
+                  targets,
```

Comment:
Can you plz explain what the additional param target does.

Reply:
yes please add something to the docstring :+1: